content/enterprise/release/index.mdx

```diff
@@ -7,30 +7,15 @@ description: >-
-| [v202106-1](/enterprise/release/v202106-1)   | 544              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/) | [1.0.0](https://github.com/hashicorp/terraform/releases/tag/v1.0.0)   | [0.18.3](https://docs.hashicorp.com/sentinel/changelog#0-18-3-june-1-2021)     |
-| [v202105-1](/enterprise/release/v202105-1)   | 534              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/) | [0.15.3](https://github.com/hashicorp/terraform/releases/tag/v0.15.3) | [0.18.1](https://docs.hashicorp.com/sentinel/changelog#0-18-1-may-11-2021)     |
-| [v202104-1](/enterprise/release/v202104-1)   | 528              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/) | [0.15.0](https://github.com/hashicorp/terraform/releases/tag/v0.15.0) | [0.18.0](https://docs.hashicorp.com/sentinel/changelog#0-18-0-march-25-2021)   |
-| [v202103-3](/enterprise/release/v202103-3)   | 523              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.14.7](https://github.com/hashicorp/terraform/releases/tag/v0.14.7) | [0.17.4](https://docs.hashicorp.com/sentinel/changelog#0-17-4-february-2-2021) |
-| [v202103-2](/enterprise/release/v202103-2)   | 520              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.14.7](https://github.com/hashicorp/terraform/releases/tag/v0.14.7) | [0.17.4](https://docs.hashicorp.com/sentinel/changelog#0-17-4-february-2-2021) |
-| [v202103-1](/enterprise/release/v202103-1)\* | 519              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.14.7](https://github.com/hashicorp/terraform/releases/tag/v0.14.7) | [0.17.4](https://docs.hashicorp.com/sentinel/changelog#0-17-4-february-2-2021) |
-| [v202102-2](/enterprise/release/v202102-2)   | 509              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5) | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
-| [v202102-1](/enterprise/release/v202102-1)   | 507              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5) | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
-| [v202101-1](/enterprise/release/v202101-1)   | 504              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/) | [0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5) | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
+A new Terraform Enterprise version is released each month. The releases from the current calendar year, plus the last
+required release, are listed in the table below. Previous releases can be found on the sidebar.
```

Comment:
I'm assuming we are only listing the ones that are still supported, yes? Push back if I'm wrong :p 

```suggestion
required release, are listed in the table below. You can find previous supported releases in the sidebar.
```